### PR TITLE
Option to disable removal of deployments

### PR DESCRIPTION
**Pull request by ProxieTV**
Since CloudNet clears the deployments into the service after deploying its ressources, in some cases it would be helpful having the ability to disable this.

An example usecase: Sometimes services have to be copied during runtime without stopping them, for example to copy data securely to prefent data loosing in crash cases.

Added overloaded methods for fully api-support

### CODE REVIEW

**Comment by realPanamo** [comment]
You can always copy services at runtime, and there was also a copy command added here #52

**Comment by realPanamo** [comment]
I dont see an advantage when being able to disable this

File: cloudnet/src/main/java/de/dytanic/cloudnet/service/JVMCloudService.java [hunk 1]
```diff
@@ -532,7 +532,7 @@ public void deployResources() {
                             deployment.getTemplate()
                     );
 
-                    this.deployments.remove(deployment);
+                    if (removeDeployments) this.deployments.remove(deployment);
```
Comment (realPanamo) [inline cloudnet/src/main/java/de/dytanic/cloudnet/service/JVMCloudService.java]: Please don't use one-line if-statements

**Comment by ProxieTV** [comment]
By using the copy command your software won't know when the copy operation is finished, so in some cases impossible to use without ugly delayed task (you don't know how long to wait, so to be sure that costs time)

So actually you have to add every deployment again after deploy ressources, that's no good solution.

**Comment by realPanamo** [comment]
I just noticed the changes are only in the api, its fine then. But please correct your changes

File: cloudnet/src/main/java/de/dytanic/cloudnet/service/JVMCloudService.java (hunk 1 above)
Comment (ProxieTV) [reply cloudnet/src/main/java/de/dytanic/cloudnet/service/JVMCloudService.java]: i'm sorry, wrapper using comes from Kotlin development :D

**Comment by ProxieTV** [comment]
yes i'll do that

**Comment by ProxieTV** [comment]
done.

**Pull request merged**